how-to-guides/12-param-schedulers.ipynb

```diff
@@ -0,0 +1,835 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+        "<!-- ---\n",
+        "title: How to use ParamScheduler with Ignite\n",
+        "date: 2022-03-17\n",
+        "downloads: true\n",
+        "weight: 12 \n",
+        "tags:\n",
+        "  - parameter schedulers\n",
+        "  - param schedulers\n",
+        "--- -->\n",
+        "\n",
+        "# How to use ParamScheduler with Ignite\n",
+        "\n",
+        "This how-to-guide aims at presenting the use of ignite parameter schedulers and advantages over the default ones.\n",
+        "\n",
+        "See the [PyTorch-Ignite implementation](https://github.com/pytorch/ignite/blob/master/ignite/handlers/param_scheduler.py) and [documentation](https://pytorch.org/ignite/master/contrib/handlers.html#module-ignite.handlers.param_scheduler) as well as the [PyTorch documentation](https://pytorch.org/docs/stable/optim.html#how-to-adjust-learning-rate).\n",
+        " "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+        "import ast\n",
```

Comment:
Useful ?

Reply:
It is used in `MultiStepLR`  for parsing the string values of Milestones.